packages/contracts/contracts/L2/ValidatorRewardVault.sol

```diff
@@ -122,6 +123,24 @@ contract ValidatorRewardVault is FeeVault, Semver {
         );
     }
 
+    /**
+     * @notice Withdraws all of the sender's balance to L2.
+     */
+    function withdrawToL2() external {
+        uint256 balance = rewards[msg.sender];
```

Comment:
I wonder why not checking if the balance is greater than `MIN_WITHDRAWAL_AMOUNT`?

Reply:
IMO, the reason why we don't need this check is because the `MIN_WITHDRAWAL_AMOUNT` is 0, not transaction fee. If I'm right, I also think that it's better to check the withdrawal amount for consistency.

Reply:
`MIN_WITHDRAWAL_AMOUNT` could be a value greater than 0 in the future, so if we decide to check this condition to restrict all withdrawals less than a threshold, it should be added. Otherwise we decide to check this condition because withdrawal fee to L1 is expensive, it could not be added. IMO, the previous semantic seems to be more proper, wdyt?

Reply:
Oh I see the changes after this comment. I'll check it!